_webi/template.sh

```diff
@@ -598,7 +604,9 @@ __bootstrap_webi() {
 
         webi_link
 
-        _webi_enable_exec
+        if test -z "${pkg_no_exec}"; then
```

Comment:
this allows us to skip `xattr` no git cloned directories, which aren't handled properly by `xattr`.